nox.py

```diff
@@ -20,8 +20,8 @@
 ON_APPVEYOR = os.environ.get('APPVEYOR') == 'True'
 
 
-@nox.session
-def default(session):
+@nox.session(python=['3.4', '3.5', '3.6'])
```

Comment:
I thought you support 3.5+ now; this should be `['3.5', '3.6', '3.7']`, yes?
(I care more about the presence of 3.7 than absence of 3.4.)

Reply:
Done